<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 213, 218);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>5</number>
      </property>
      <property name="text">
       <string>==ADMIN==</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUnblock">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="text">
       <string>Unblock an account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRegister">
      <property name="text">
       <string>Create an Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCustomer">
      <property name="text">
       <string>View Customer History</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnUser">
      <property name="text">
       <string>Update/Check An Account</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHistory">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="text">
       <string>View Overall ATM HISTORY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(172, 125, 136);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
